app/models/visit.rb

```diff
@@ -18,7 +18,10 @@ class Visit
   }
   validate :validate_amount_of_adults, on: :visitors_set
 
-  delegate :prison_name, :prison_name=, to: :prisoner, allow_nil: true
+  delegate :prison, :prison_name, :prison_name=,
+    to: :prisoner, allow_nil: true
+  delegate :prison_email, :prison_canned_responses, :prison_nomis_id,
+    to: :prisoner
 
   def validate_amount_of_adults
     if visitors.none? { |v| v.age && v.age >= 18 }
```

Comment:
Regarding the magic number 18 here.. shouldn't this be using the prisons adult age?